### ISSUE TITLE
High 'level' pokemon transferred

### ISSUE THREAD

**Issue by vison99** [PocketMobsters/PokeMobBot#411]
As you know each Pokemon has their own level i.e. 30, 30.5, 31, for upgrade purposes. You always want to capture the highest possible level (based on your character level), because if they are useful and evolved, can become powerful and save you tons of stardust and candies. I am talking about 100-200k of stardust saved.

I could be wrong, but my understand is that the Bot currently do not give regards to Pokemon level.

For example, my Bot just transferred an almost maxed level (for my level) CP 900 Exeggcute because its IV were 79% and decided to keep the Exeggcute that has CP 200 and IV 81%. If I evolve this transferred Exeggcute, it will be stronger than my current Exeggcute, and possible offer a different move set, unfortunately it was transferred.

Not saying 79% IV is perfect and worth keeping, but it can certainly improve my current gym team and save me tons of stardust. The same has been happening for lots of other Pokemons with high CP and level but slightly lower IV.

I guess I am not the only one frustrated and disappointed to have transferred so many great catches. I know we shouldn't keep too many duplicates but I feel there is a need to introduce a 'level keep' control. i.e. keep all/particular level xx.x Pokemon. Or alternatively, allow more complicated global settings. It can be a 'true/false' settings for sophisticated players. The goal is to keep high level Pokemon for evolution and move set purposes.

So far, the benefits of IV is far less than move sets. Having multiple high level Pokemon is better having a perfect IV with a fixed move set. Please let me know what you think.

On a side note, I am still running 1.0. As 1.01 had a few reliability issues so I decided to go back. The original version has been running flawless over the last couple of days.

Of course, thanks for producing such a quality work. Well done!

**Comment by vison99** [comment]
Sorry my mistake. The bot does read the level in catches, but is there a way to use them in the config?

**Comment by eslindsey** [comment]
See `"PrioritizeIvOverCp": true,` in the `config.json` file. Set it to FALSE if you want the bot to use CP when determining which Pokemon to transfer.

**Comment by vison99** [comment]
Thanks Eslindsey.

I dont want high CP. I want high level. As low CP high level ones can evolve into high CP. i.e. for example a 180CP Magikarp is more useful than a 1200 bat.

I have enough high CP useless, evolved pokemons, I now set global minimum to 1500.

**Comment by Kureka** [comment]
I agree with you Vison99, I also want to keep the highest level as priority (31 in this case) with best moveset as 2nd priority, IV as 3rd priority and CP as 4th priority if possible.

**Comment by tangdami** [comment]
I totally agree with you! I have been asking this option (keep pokemons by using their levels not CP). But it seems not many people care that. My current level is 32, I would love to keep any pokemon with LEVEL 30 to 32 because they always have the best CP if they have a decent IV.

**Comment by Kalashnikitty** [comment]
yes, this option would be usefull.

**Comment by vapp13** [comment]
+1 I support this request too.
